### PR TITLE
Fix #835 GitHub login for community pages

### LINKED ISSUE #835: Github login doesn't work for community pages
To add RSS feeds, you need to be logged in and you are redirected to this [URL](https://www.djangoproject.com/community/add/blogs/). And if you go to the signup page, you can login through Github. After you do that and try to add the RSS feed, you are again redirected to login page. So, it's not creating an account like the normal signup process does.

**Take note of the starting URL**.

Here is a GIF of the entire experience:

djangoproject

More Info: 
Fedora 27
Tested on Firefox developer edition and Chrome

### PR DESCRIPTION
**Pull request by Sanji515**

### CODE REVIEW

**Comment by Sanji515** [comment]
@timgraham Please review my Pull request.

File: djangoproject/settings/common.py [hunk 1]
```diff
@@ -206,11 +207,23 @@
...
             ],
         },
     },
 ]
 
+AUTHENTICATION_BACKENDS = (
+    'social_core.backends.open_id.OpenIdAuth',
+    'social_core.backends.google.GoogleOpenId',
+    'social_core.backends.google.GoogleOAuth2',
+    'social_core.backends.github.GithubOAuth2',
```
Comment (Sanji515) [inline djangoproject/settings/common.py]: @timgraham Actually here only `GithubOAuth2` is used and rest are of no use, I just pasted from my previous project

File: djangoproject/templates/registration/registration_form.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 {% block content %}
   <h1>Create an account</h1>
 
-  <p>Or, <a href="https://code.djangoproject.com/github/login">login with GitHub</a>.</p>
+  <p>Or, <a href="{% url 'social:begin' 'github' %}">login with GitHub</a>.</p>
```
Comment (Sanji515) [inline djangoproject/templates/registration/registration_form.html]: This is where I removed the URL of redirecting again to the login page.

**Comment by timgraham** [comment]
I'm not sure what the solution is but I don't think think this on the right track.

**Pull request closed without merging**